gradle/libs.versions.toml

```diff
@@ -0,0 +1,70 @@
+aws-sdk-glue = { module = "software.amazon.awssdk:glue", version.ref = "aws" }
+aws-sdk-kms = { module = "software.amazon.awssdk:kms", version.ref = "aws" }
+aws-sdk-lakeformation = { module = "software.amazon.awssdk:lakeformation", version.ref = "aws" }
+aws-sdk-sts = { module = "software.amazon.awssdk:sts", version.ref = "aws" }
+aws-sdk-s3 = { module = "software.amazon.awssdk:s3", version.ref = "aws" }
+aws-sdk-url-connection-client = { module = "software.amazon.awssdk:url-connection-client", version.ref = "aws" }
+checkstyle = { module = "com.puppycrawl.tools:checkstyle", version.ref = "checkstyle" }
+errorprone-annotations = { module = "com.google.errorprone:error_prone_annotations", version.ref = "errorprone" }
+errorprone-core = { module = "com.google.errorprone:error_prone_core", version.ref = "errorprone" }
+errorprone-slf4j = { module = "jp.skypencil.errorprone.slf4j:errorprone-slf4j", version.ref = "errorproneSlf4j" }
+ecs-bundle = { module = "com.emc.ecs:object-client-bundle", version.ref = "ecs" }
+findbugs-annotations = { module = "com.google.code.findbugs:annotations", version = "3.0.1" }
+findbugs-jsr305 = { module = "com.google.code.findbugs:jsr305", version = "3.0.2" }
+google-java-format = { module = "com.google.googlejavaformat:google-java-format", version.ref = "googleJavaFormat" }
+guava = { module = "com.google.guava:guava", version.ref = "guava" }
+hadoop-aws = { module = "org.apache.hadoop:hadoop-aws", version.ref = "hadoop" }
+hadoop-common = { module = "org.apache.hadoop:hadoop-common", version.ref = "hadoop" }
+iceberg-dell = { module = "org.apache.iceberg:iceberg-dell", version.ref = "iceberg" }
+iceberg-spark-runtime = { module = "org.apache.iceberg:iceberg-spark-runtime-3.3_2.12", version.ref = "iceberg" }
+immutables-builder = { module = "org.immutables:builder", version.ref = "immutables" }
+immutables-value-annotations = { module = "org.immutables:value-annotations", version.ref = "immutables" }
+immutables-value-processor = { module = "org.immutables:value-processor", version.ref = "immutables" }
+jacoco-ant = { module = "org.jacoco:org.jacoco.ant", version.ref = "jacoco" }
+jacoco-report = { module = "org.jacoco:org.jacoco.report", version.ref = "jacoco" }
+jacoco-maven-plugin = { module = "org.jacoco:jacoco-maven-plugin", version.ref = "jacoco" }
+jandex = { module = "org.jboss:jandex", version.ref = "jandex" }
+junit-bom = { module = "org.junit:junit-bom", version.ref = "junit" }
+junit-jupiter-api = { module = "org.junit.jupiter:junit-jupiter-api", version.ref = "junit" }
+junit-jupiter-engine = { module = "org.junit.jupiter:junit-jupiter-engine" }
+junit-jupiter-params = { module = "org.junit.jupiter:junit-jupiter-params" }
+logback-classic = { module = "ch.qos.logback:logback-classic", version.ref = "logback" }
+logcaptor = { module = "io.github.hakky54:logcaptor", version.ref = "logcaptor" }
+mysql-driver = { module = "mysql:mysql-connector-java", version.ref = "mysqlDriver" }
```

Comment:
GPL2 w/ exceptions licensed, NOT ASF2 licensed.
Remove it from this PR.
I do not wanna go down the rabbit hole and discuss this issue in this PR.

Reply:
removed. 

User can add this to the classpath during migration to/from JDBC catalog (backed by MYSQL)